__tests__/SetsSelectors.test.js

```diff
@@ -899,6 +899,7 @@ describe('SetsSelectors', () => {
                         removed: true,
                         data: [-3456, 37, 1.453368, 328, 83, 72, 11, 15]
                     }],
+                    initialStartTime: '2017-12-30T04:06:12.640Z'
```

Comment:
Out of curiosity, does making it a real date work as well?
 
- [ ] check that it works with real dates in addition to string dates